lib/docker.js

```diff
@@ -464,16 +464,17 @@ Docker.prototype.getEvents = function(opts, callback) {
  * @param  {String}   repoTag  Repository tag
  * @param  {Object}   opts     Options (optional)
  * @param  {Function} callback Callback
+ * @param  {Object}   auth     Authentication (optional)
  * @return {Object}            Image
  */
-Docker.prototype.pull = function(repoTag, opts, callback) {
+Docker.prototype.pull = function(repoTag, opts, callback, auth) {
```

Comment:
Would it be possible to add auth to opts instead? It's really unusual in node to have arguments after the callback.